ISLE/res/arrow_bmp.h

```diff
@@ -0,0 +1,356 @@
+unsigned char arrow_bmp[] = {
```

Comment:
Needed for `SDL_IOFromConstMem`, also for other headers.
```suggestion
const unsigned char arrow_bmp[] = {
```